### ISSUE TITLE
Help with implementing classes

### ISSUE THREAD

**Issue by pintariching** [pintariching/railwind#2]
Using the currently implemented classes as a example, I would need help to speed up the process of implementing the rest of tailwinds classes.

- [x] Sizing
- [x] Typography
- [x] Backgrounds
- [x] Borders
- [x] Effects
- [x] Filters
- [x] Tables
- [x] Transitions & Animations
- [x] Transforms
- [x] Interactivity
- [x] SVG
- [x] Accessibility

**Comment by vermutely** [comment]
I am new to Rust and would like to support this project.
I am checking out the past commits right now. I would like to help add a class, are there any rules?

**Comment by pintariching** [comment]
Hey! No I don't have any special rules just choose a class that hasn't been yet implemented and look copy the implementation of other classes.

For classes that have to support theme customization, I load the values from a `.ron` file into a `HashMap` so it can later be extended with custom values.

Also if you want, you can also add more test cases, I think I'm not covering all the implemented classes yet. For that, I copy all the classes into [play.tailwind.com](https://play.tailwindcss.com/) and copy the output into an `expected.css` and the input into `input.html` and compare the `output.css` with the expected and see what should be changed.

**Comment by vermutely** [comment]
@pintariching 
Thanks for confirming!
I will try to implement Transforms. I will create a pull request once the implementation is finished.
I'll try to challenge you on the test code, etc., so please be patient!

**Comment by pintariching** [comment]
No worries, you can also open a half completed pull request and I can guide you how I've implemented stuff if you need help!

**Comment by vermutely** [comment]
@pintariching 
I have created a pull request for Transforms( #5 ). I will go into the implementation of Transitions & Animations next.

**Comment by pintariching** [comment]
@vermutely Thanks! I'll add the tests and see if everything works.

**Comment by vermutely** [comment]
@pintariching 
Thanks all for the lighting fast response!
I'm thinking of implementing **Filters** and **Tables** next, does the work conflict?

**Comment by pintariching** [comment]
@vermutely No currently I'm not working on anything, feel free to open a PR

**Comment by vermutely** [comment]
@pintariching 
Thanks for confirming.

I will gradually increase my class support.
I will try to write to this Issue before I start working on other classes so that others can also help.

**Comment by vermutely** [comment]
I have submitted a pull request for Tables and Filters, so the next step is to implement Interactivity.

**Comment by vermutely** [comment]
I have submitted a pull request for Interactivity.
SVG and Accessibility next.

**Comment by vermutely** [comment]
I have submitted a pull request for SVG and Accessibility. (Thank you review!!)  
Is this tasks complete?

**Comment by pintariching** [comment]
Wow, I didn't expect it to be done this quickly, thank you for everything @vermutely !

Now there's a few tasks of the top of my head that can be done at once I think.

- The CLI needs to be improved to support customizability and to have it watch folders,
- I would like to have more tests, for every class, to confirm everything works
- Make packages for NPM, Deno, add Trunk support

I'll open a new issue with everything listed later today
